src/traces/image/attributes.js

```diff
@@ -84,13 +84,13 @@ module.exports = extendFlat({
         valType: 'any',
         dflt: 0,
         editType: 'calc+clearAxisTypes',
-        description: 'Set the image\'s x position.'
+        description: 'Set the image\'s x position. The bottom left corner of the image will be found at xmin=x0-dx/2'
```

Comment:
This depends on the `xaxis.range`. For *reversed* axes the opposite is true.

Reply:
Do you mean that in reversed axis the bottom left corner would be at `xmax=x0+dx/2`?

    Set the image\'s x position. In absence of axis transformations, the bottom left corner of the image will be found at `xmin=x0-dx/2`

Would that be a more accurate statement?

Reply:
Thanks @zeehio - I think this is the complete condition (and yes, it does work to set a negative `dx`, I just tried it 😅):
```suggestion
        description: [
            'Set the image\'s x position. The left edge of the image',
            '(or the right edge if the x axis is reversed or dx is negative)',
            'will be found at xmin=x0-dx/2'
        ].join(' ')
```

Reply:
Actually negative `dx` doesn't really work right either... so maybe unless (or until) we want to fix that we should force `dx` and `dy` to be positive.

Reply:
https://github.com/plotly/plotly.js/issues/6600